src/web/views/enter-name.njk

```diff
@@ -12,7 +13,8 @@
     }
   }) %}
 
-    <p class="govuk-body">{{ formDescription }}</p>
+    <br>
```

Comment:
Is this `<br />` for spacing? It would be better to use a class on the component above.

Reply:
Space no longer required so removed.